package.json

```diff
@@ -172,10 +172,12 @@
     "scripts/**/*",
     "scss/**/*",
     "src/**/*",
-    "umd/**/*"
+    "umd/**/*",
+    "!**/*.config.js",
+    "!**/__tests__/**/*"
   ],
   "scripts": {
-    "build": "gulp build",
+    "build": "gulp build && gulp html:source -e",
```

Comment:
Shouldn't be necessary as long as we flip feature flags, but definitely let me know if you find otherwise!

Reply:
@asudoh it seems like the default build doesn't generate this, unfortunately. Seems like something is going on 🤔

Reply:
OK good catch @joshblack! Created: #2215